site/docs/configuration.md

```diff
@@ -81,6 +81,8 @@ Iceberg tables support table properties to configure table behavior, like the de
 | commit.manifest.target-size-bytes  | 8388608 (8 MB)   | Target size when merging manifest files                       |
 | commit.manifest.min-count-to-merge | 100              | Minimum number of manifests to accumulate before merging      |
 | commit.manifest-merge.enabled      | true             | Controls whether to automatically merge manifests on writes   |
+| history.expire.max-snapshot-age-ms | 432000000 (5 days) | Default max age of snapshots to keep while expiring snapshots    |
+| history.expire.retain-min-num-snapshots | 1             | Default min number of snapshots to keep while expiring snapshots |
```

Comment:
What do you think about a slight rename so that `min` comes first and we remove `num`? I'm thinking `history.expire.min-retained-snapshots`. We could also use `kept` instead of `retained` if that is more clear.